src/kakarot/instructions/environmental_information.cairo

```diff
@@ -549,7 +549,7 @@ namespace EnvironmentalInformation {
         bitwise_ptr: BitwiseBuiltin*,
     }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
         // Get return data size.
-        let return_data_size = Helpers.to_uint256(ctx.sub_context.return_data_len);
+        let return_data_size = Helpers.to_uint256(ctx.return_data_len);
```

Comment:
my updated understanding of the topic is that there is only one RETURN_DATA storage, and that the previous implementation was somehow a mistake. Since the current behavior is the same as in geth and in the execution specs, i think that it's fine like that